metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -355,6 +347,9 @@ class MetalsLanguageServer(
   @JsonNotification("exit")
   def exit(): Unit = {
     shutdown()
```

Comment:
Is this call idepmpotent? Calling shutdown then exit will go through shutdown twice.

Reply:
Yes it is, added a comment to explain why.